templates/documents/general_list.html

```diff
@@ -41,7 +41,14 @@
 <script type="text/javascript" src="{$GLOBALS.assets_static_relative}/dropzone/dist/dropzone.js"></script>
 
 <script type="text/javascript">
-    // dropzone javascript asset translation(s)
+	function callTemplateModule() {literal}{{/literal}
+		top.restoreSession();
+		let tele  = document.getElementById("template_filename");
+		let tname = encodeURIComponent(tele.options[tele.selectedIndex].value);
+		let callUrl = '{$GLOBALS.webroot}/portal/patient/onsitedocuments?pid={$patient_id|attr}&catid={$category_id|attr}&is_module=true&new=' + tname;
```

Comment:
oh, nothing like some smarty code :)
below will better escape in this case (using the infamous js_url function :) ):
```
let callUrl = '{$GLOBALS.webroot}/portal/patient/onsitedocuments?pid=' + {$patient_id|js_url} + '&catid=' + {$category_id|js_url} + '&is_module=true&new=' + tname;
```